### PR TITLE
Lesson 3 part 2: Bash Script

### PR DESCRIPTION
**Pull request by vizhyk**
Done!
Please, review my task.

### CODE REVIEW

**Comment by vizhyk** [comment]
@a-vodka Please check it.

File: script.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+!#/bin/bash
+find /home/vvi/part1 -name "*.[ch]*" -exec cp {} /tmp/guesanumber/  \; || echo "no files"
```
Comment (a-vodka) [inline script.sh]: Please define /tmp/guesanumber/ as variable.

File: script.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+tar cfz /tmp/guesanumber/arc.tgz /tmp/guesanumber/*
+mv /tmp/guesanumber/arc.tgz /home/vvi/part1/release
```
Comment (a-vodka) [inline script.sh]: hard-coding /home/vvi/part1/release is bad idea. Please use relative path instead.

**Comment by vizhyk** [comment]
@a-vodka  Now it's perfect(I HOPE).

**Pull request merged**